dockstore-webservice/src/main/resources/migrations.1.5.0.xml

```diff
@@ -30,4 +30,44 @@
         <addNotNullConstraint columnDataType="clob" columnName="referencetype" tableName="tag"/>
         <addNotNullConstraint columnDataType="clob" columnName="referencetype" tableName="workflowversion"/>
     </changeSet>
+
+    <!-- adding support for input and output format -->
+    <changeSet author="gluu (generated)" id="1525277022753-5">
+        <createTable tableName="fileformat">
+            <column autoIncrement="true" name="id" type="BIGSERIAL">
+                <constraints primaryKey="true" primaryKeyName="fileformat_pkey"/>
+            </column>
+            <column name="value" type="VARCHAR(255)"/>
```

Comment:
Just use text here, there's no performance benefits with limited varchars  
https://www.postgresql.org/docs/9.1/static/datatype-character.html

Reply:
this is generated, are you sure?

Reply:
Yes, checkout the tip

Reply:
I think the tip just says there's no difference.